setup.py

```diff
@@ -142,7 +142,6 @@ def readfile(fname):
         'Hierarchical Dirichlet Process, HDP, Random Projections, '
         'TFIDF, word2vec',
 
-    license='LGPL',
```

Comment:
OK. Do we want to support v2 though? I've never heard of that version, it's another potential can of worms. I'd prefer 2.1+.